tests/gentype_tests/typescript-react-example/package-lock.json

```diff
@@ -21,6 +21,7 @@
       }
     },
     "../../..": {
+      "name": "rescript",
```

Comment:
Not sure why this keeps changing back and forth.
Probably depending on the npm version used?
If we want this change, we should do it in a separate PR for all package-lock.json files.

Reply:
Removed